app/controllers/hub/organizations_controller.rb

```diff
@@ -38,18 +38,20 @@ def edit
 
     def update
       if @vita_partner.update(vita_partner_params)
+        flash[:notice] = I18n.t("general.changes_saved")
```

Comment:
51% of me thinks it's great that you're adding this flash notice. 49% of me thinks we should do it on every form, or on no forms, to help our users have consistent expectations.

I think this form is unusual in that the "update" goes back to "edit", rather than to "show", and that justifies the unusual flash notices.